tool_collections/gops/basecoverage/operation_filter.py

```diff
@@ -1,8 +1,5 @@
 # runs after the job (and after the default post-filter)
-import os
-from galaxy import eggs
-from galaxy import jobs
-from galaxy.tools.parameters import DataToolParameter
+from galaxy.tools.parameters.basic import DataToolParameter
```

Comment:
No, it is imported only in gops tools. Anyway, I can revert this change if you prefer.